styles.css

```diff
@@ -111,3 +111,129 @@
 .main__button:hover {
   opacity: .5;
 }
+
+.burger__menu {
+  display: none;
+}
+
+@media (max-width: 1560px) {
+  .wrapper {
+    width: 69%;
+  }
+
+  .menu__link {
+    font: 500 14px/16px 'Roboto', sans-serif, arial, helvetica;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6306727)